core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -203,6 +218,9 @@ public class JGroupsTransport implements Transport, ChannelListener, AddressGene
    // ------------------------------------------------------------------------------------------------------------------
    private boolean running;
 
+   // effective immutable
+   private boolean tracingEnabled;
```

Comment:
Don't we have to register a listener with the cache configuration though to detect these changes? Or was there something else added that will do this that I am not aware of?

Ah I guess I missed that there is a global enabled and a cache based enabled in addition to all the categories, have to admit it is a bit confusing. Maybe the cache based one should be named something different than just enabled since we have the categories now?

Reply:
@wburns Sure let's find another name. Maybe we can call it `cacheEnabled`?